pom.xml

```diff
@@ -67,97 +67,108 @@ of Jackson components maintained by FasterXML.com
     <jackson.version.jacksonjr>${jackson.version}</jackson.version.jacksonjr>
 
     <jackson.version.module>${jackson.version}</jackson.version.module>
-    <jackson.version.module.kotlin>${jackson.version.module}</jackson.version.module.kotlin>
-    <jackson.version.module.scala>${jackson.version.module}</jackson.version.module.scala>
+    <jackson.version.module.kotlin>${jackson.version}</jackson.version.module.kotlin>
+    <jackson.version.module.scala>${jackson.version}</jackson.version.module.scala>
     <!-- JPMS Library Updates-->
     <javax.activation.version>1.2.0</javax.activation.version>
 
     <!-- for Reproducible Builds -->
     <project.build.outputTimestamp>2024-11-01T19:43:09Z</project.build.outputTimestamp>
   </properties>
 
+  <!-- 31-Jan-2025, elect86: I flattened the properties, because Gradle limitations
+        Gradle can resolve this:
```

Comment:
Thank you for adding this explanation! I have one question based on it, bit further down, will add it there.